ui/tests/unit/services/breadcrumbs-test.js

```diff
@@ -14,15 +14,23 @@ const makeRoute = (crumbs, controller = {}) =>
     controller: Controller.extend(controller).create(),
   });
 
-moduleFor('service:breadcrumbs', 'Unit | Service | Breadcrumbs', {
-  beforeEach() {
+module('Unit | Service | Breadcrumbs', function(hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.subject = function() {
+      return this.owner.factoryFor('service:breadcrumbs').create();
```

Comment:
This boilerplate for unit testing a service is kind of a bummer, but understandable.

Reply:
There may be a better way to do this, such as something like `setupSerializerTest(hooks)`, but I don't want to explore that until I make it to v3.4.